src/Microsoft.Framework.Caching.Abstractions/project.json

```diff
@@ -1,6 +1,10 @@
 {
     "dependencies": {
-        "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" }
+      "Microsoft.Framework.NotNullAttribute.Sources": {
+        "type": "build",
+        "version": "1.0.0-*"
+      },
+      "Newtonsoft.Json": "6.0.6"
```

Comment:
I don't think we want this in the box. Better to start this off as an extension to see how people use the API.